mlflow/pyfunc/__init__.py

```diff
@@ -141,6 +143,127 @@ def add_to_model(model, loader_module, data=None, code=None, env=None, **kwargs)
     return model.add_flavor(FLAVOR_NAME, **parms)
 
 
+def save_model(path, model_class, artifacts=None, parameters=None, conda_env=None, code_paths=None,
```

Comment:
There is a change in signature. That would be a problem.

Reply:
I've implemented a backwards-compatible API. Please take a look!